### DISCUSSION TITLE
Automated this thing

### DISCUSSION THREAD

**Discussion by Atrone** [kyleskom/NBA-Machine-Learning-Sports-Betting#336] [Show and tell]
"is my data correct?"
"what's the stats on this, currently?"
"how the do I use this?"

Look no further.

https://github.com/Atrone/NBA-Machine-Learning-Sports-Betting

Not sure if adding days rest will improve or confuse the models, but I'm doing good so far!

**Comment by kyleskom** [comment]
Open a PR!

**Comment by STRATZ-Ken** [comment]
I vote we ban him and remove this post.

Sent from my iPhone

On Nov 5, 2023, at 6:25 PM, Anthony Teixeira ***@***.***> wrote:

﻿

Hmm. I want to monetize my work tho. (No user pass without some dollers every season)

**Reply by Atrone** [reply]
Why?

**Reply by STRATZ-Ken** [reply]
@kyleskom I suggest removing this asap.  The toxic nature of this guy is a joke.  Take your free work, and then trying to sell it as his own.  I would also ban him from the repo as well.

**Comment by Emillzz45** [comment]
I've been following this for years and never seen this 👀
